proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -157,3 +164,23 @@ message QueryDenomMetadataResponse {
   // metadata describes and provides all the client information for the requested token.
   Metadata metadata = 1 [(gogoproto.nullable) = false];
 }
+
+// QueryDenomOwnersRequest defines the request type for the DenomOwners RPC query,
+// which queries for a paginated set of all account holders of a particular
+// denomination.
+message QueryDenomOwnersRequest {
+  // denom defines the coin denomination to query all account holders for.
+  string denom = 1;
+
+  // pagination defines an optional pagination for the request.
+  cosmos.base.query.v1beta1.PageRequest pagination = 2;
+}
+
+// QueryDenomOwnersResponse defines the RPC response of a DenomOwners RPC query.
+message QueryDenomOwnersResponse {
+  // addresses defines the set of addresses that own a particular denomination.
+  repeated string addresses = 1;
```

Comment:
do we want to include balances here too?

Reply:
I think that'd be useful too. @alexanderbez ?

Reply:
Sure, I'll make this change.

Reply:
Done :)